### ISSUE TITLE
about the shape of A

### ISSUE THREAD

**Issue by pandaQJY** [liangzhehan/DMSTGCN#3]
Hello, I saw that the shapes of tensor x and A are ncvl and nwv in the model.py. Do you mean that the two “n” are equal and n is equal to batch size? However, the data of the same batch may not always be in several same time slots. Didn't you shuffle the data before training？This part is inconsistent with the papers, I hope you can answer the puzzle.

**Comment by liangzhehan** [comment]
Yes, the "n" is batch size. If data of the same batch is not in same time slots, the tensor A would contain dynamic graphs in these different time slots respectively. For example if there are data at 8:00 and 9:00, the a slice of A would be graph at 8:00 and another slice would be graph at 9:00. These two samples in the same batch would be computed in parallel.
